src/client/src/app/site/[site]/cancel-day/cancel-day-form.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { screen } from '@testing-library/react';
+import CancelDayForm from './cancel-day-form';
+import render from '@testing/render';
+import { useRouter } from 'next/navigation';
+import { parseToUkDatetime } from '@services/timeService';
+import * as appointmentsService from '@services/appointmentsService';
+
+jest.mock('next/navigation', () => ({
+  useRouter: jest.fn(),
+}));
+
+jest.mock('@services/timeService', () => ({
+  parseToUkDatetime: jest.fn(),
+}));
+
+jest.mock('@components/session-summary-table', () => ({
+  SessionSummaryTable: ({ tableCaption }: { tableCaption: string }) => (
+    <div data-testid="session-summary">{tableCaption}</div>
```

Comment:
Whilst it can sometimes be necessary, using `data-testid` is generally considered a code smell because if you can't find an element without using its id then that element is probably not accessible. 

In this case, it's totally redundant as the below assertion which uses it can just be rewritten as 
<img width="924" height="195" alt="image" src="https://github.com/user-attachments/assets/b0f39293-22dc-4572-a4a9-fcbe0788ee9b" />
then the data-testid can be removed